config/config.json

```
@@ -27,13 +28,15 @@
         "alpha.cplusplus.DeleteWithNonVirtualDtor" : ["default", "sensitive", "extreme"],
         "alpha.cplusplus.IteratorRange" : [],
         "alpha.cplusplus.MisusedMovedObject" : ["sensitive", "extreme"],
-        "alpha.security.ArrayBound" : ["extreme"],
-        "alpha.security.ArrayBoundV2" : ["extreme"],
-        "alpha.security.MallocOverflow" : ["sensitive", "extreme"],
-        "alpha.security.ReturnPtrRange" : ["sensitive", "extreme"],
-        "alpha.security.taint.TaintPropagation" : ["sensitive", "extreme"],
+        "alpha.cplusplus.UninitializedObject" : ["default", "sensitive", "extreme"],
+        "alpha.security.ArrayBound" : ["extreme","security"],
```

Please put a space between a comma and a quotation mark in every place similar to this:
```suggestion
        "alpha.security.ArrayBound" : ["extreme", "security"],
```